lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_firebase_test.rb

```diff
@@ -11,8 +11,10 @@ class AndroidFirebaseTestAction < Action
       def self.run(params)
         validate_options(params)
 
+        UI.user_error!('You must be logged in to Firebase prior to calling this action. Use the `FirebaseLogin` Action to log in if needed.') unless Fastlane::FirebaseAccount.authenticated?
```

Comment:
Nitpick: Have you considered writing the action in the syntax a user would use to call it from the CLI, which is where this error would pop up?

Reply:
This is a good idea – there's more area for improvement here for sure (for one, right now there's currently only one way to call this action, but `gcloud` supports a few different authentication mechanisms). 

I'd like to land this PR to unblock some rather urgent fixes, but there's a bunch of other work we can do in this area for sure! (see the `add/gradle-output-parsing` branch for one such example)